.github/scripts/trymerge.py

```diff
@@ -1265,6 +1321,86 @@ def checks_to_str(checks: List[Tuple[str, Optional[str]]]) -> str:
 def checks_to_markdown_bullets(checks: List[Tuple[str, Optional[str]]]) -> List[str]:
     return [f"- [{c[0]}]({c[1]})" if c[1] is not None else f"- {c[0]}" for c in checks[:5]]
 
+def get_flaky_rules(url: str, num_retries: int = 3) -> List[FlakyRule]:
+    try:
+        return [FlakyRule(**rule) for rule in fetch_json_list(url)]
+    except Exception as e:
+        print(f"Could not download {url} because: {e}.")
+        if num_retries > 0:
+            return get_flaky_rules(url, num_retries=num_retries - 1)
+        return []
+
```

Comment:
Aren't we running `flake8` on this file? There should be two newlines between functions/classes (but one newline between class methods)
```suggestion


```